canonical-web-and-design/charmhub.io
# Remove featured and other filters

## Done

Removed featured and other filters

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Check that "featured" and "other" aren't in the categories list

## Issue / Card

Fixes #574
